File: android-resources/native/MainActivity.java
package app.lovable.tempokey;

import android.os.Bundle;

import com.getcapacitor.BridgeActivity;

public class MainActivity extends BridgeActivity {
    @Override
    public void onCreate(Bundle savedInstanceState) {
        registerPlugin(TempoKeyPermissions.class);
        registerPlugin(TempoKeyFolderPicker.class);
        super.onCreate(savedInstanceState);
    }
}

File: android-resources/native-plugin/MainActivity.kt
package app.lovable.tempokey

import android.os.Bundle
import com.getcapacitor.BridgeActivity
import app.lovable.tempokey.plugins.FolderPickerPlugin

class MainActivity : BridgeActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        registerPlugin(FolderPickerPlugin::class.java)
        super.onCreate(savedInstanceState)
    }
}
